AutoRest/Generators/CSharp/CSharp/Templates/MethodTemplate.cshtml

```diff
@@ -30,17 +31,26 @@
 /// </param>
 public async Task<@(Model.OperationResponseReturnTypeString)> @(Model.Name)WithHttpMessagesAsync(@(Model.GetAsyncMethodParameterDeclaration(true)))
 {
+    @foreach (var groupParameter in Model.ParameterTemplateModels.Where(p => p.IsParameterGroup))
```

Comment:
@yugangw-msft Yes, that's already done in node and C# -- for Java, ParameterGrouping isn't fully supported yet since there isn't an Azure.Java yet, but for C# and node I've included examples:
```C#
/// <summary>
/// Post a bunch of required parameters grouped
/// </summary>
/// <param name='operations'>
/// The operations group for this extension method.
/// </param>
/// <param name='parameterGroupingPostRequiredParameters'>
/// Additional parameters for the operation
/// </param>
public static void PostRequired(this IParameterGroupingOperations operations, ParameterGroupingPostRequiredParameters parameterGroupingPostRequiredParameters)
```
I am not a node comment expert so it'd be good if you can confirm that this comment looks correct.

```node
/**
 * Post a bunch of required parameters grouped
 *
 * @param {object} parameterGroupingPostRequiredParameters Additional
 * parameters for the operation
 * 
 * @param {number} [parameterGroupingPostRequiredParameters.body]
 * 
 * @param {string} [parameterGroupingPostRequiredParameters.header]
 * 
 * @param {number} [parameterGroupingPostRequiredParameters.query] Query
 * parameter with default
 * 
 * @param {string} [parameterGroupingPostRequiredParameters.path] Path
 * parameter
 * 
 * @param {object} [options]
 *
 * @param {object} [options.customHeaders] headers that will be added to
 * request
 *
 * @param {function} callback
 *
 * @returns {function} callback(err, result, request, response)
 *
 *                      {Error}  err        - The Error object if an error occurred, null otherwise.
 *
 *                      {null} [result]   - The deserialized result object.
 *
 *                      {object} [request]  - The HTTP Request object if an error did not occur.
 *
 *                      {stream} [response] - The HTTP Response stream if an error did not occur.
 */
ParameterGrouping.prototype.postRequired = function (parameterGroupingPostRequiredParameters, options, callback) {
```